orderbook/usecase/orderbook_usecase.go

```diff
@@ -137,6 +137,63 @@ func (o *OrderbookUseCaseImpl) ProcessPool(ctx context.Context, pool sqsdomain.P
 	return nil
 }
 
+// GetActiveOrdersStream implements mvc.OrderBookUsecase.
+func (o *OrderbookUseCaseImpl) GetActiveOrdersStream(ctx context.Context, address string) <-chan orderbookdomain.ActiveOrder {
+	// Fetch orders duration
+	d := 5 * time.Second
+
+	// Result channel
+	c := make(chan orderbookdomain.ActiveOrder, 50) // buffered channel to avoid blocking
```

Comment:
Let's move these constants to the top of the file (defined in global space) and also list how these numbers were selected (state arbitrarily if so)